src/App.js

```diff
@@ -5,35 +5,26 @@ import ChairCard from './component/ChairCard/ChairCard';
 import data from './data.json';
 
 export default function App() {
-const chairsData = data;
+  const chairsData = data;
   const [showDescription, setShowDescription] = useState(false);
 
-  /* function toggleDescription(id) {
-  setChairsData(chairsData
-      .map(item1 => ({...item1, showDescription: false}))
-      .map(item2 => (item2._id === id ? {...item2, showDescription: !item2.showDescription} : item2)));
-    
-  }
-  */
-
-  const toggleDescription = index => {
-    showDescription === index ? setShowDescription(false) : setShowDescription(index);
+  const toggleDescription = id => {
+    showDescription === id ? setShowDescription(false) : setShowDescription(id);
   };
 
   return (
     <AppContainer>
       <h1>Find your perfect Design Chair</h1>
 
-      {chairsData.map((item, index) => (
+      {chairsData.map(item => (
         <ChairCard
           key={item._id}
           id={item._id}
           name={item.name}
           mainImg={item.imgUrl}
           description={item.description}
-          toggleDescription={() => toggleDescription(index)}
+          toggleDescription={() => toggleDescription(item._id)}
```

Comment:
Should this here be really `item._id` instead of `item.id` ? 🤔

Reply:
Because I named the id in the database  _id as well. I've seen this in many databases and thought its the propper way to name the id like this.